packages/pluggableWidgets/datagrid-date-filter-web/src/DatagridDateFilter.xml

```diff
@@ -43,6 +43,10 @@
                     <caption>Button caption</caption>
```

Comment:
Can we make this more precise, like `Comparison button caption`?

Reply:
Good suggestion. Fixed